src/averageAges.js

```diff
@@ -1,61 +1,50 @@
 'use strict';
 
-/**
- * Implement calculateMenAverageAge function
- *
- * Function returns average age of men in array. If `century` is specified then
- * function calculates average age only for men who died in this century
- *
- * To calculate century:
- * Divide year of person's death by 100: Math.ceil(person.died / 100)
- *
- * @param {object[]} people
- * @param {number} century - optional
- *
- * @return {number}
- */
+function calculateAverageAge(persons) {
+  const ages = persons.map(person => person.died - person.born);
+  const averageAge = ages.reduce((sum, age) =>
+    (sum + age)) / ages.length;
+
+  return averageAge;
```

Comment:
this way we shorten your code without losing any readability

and if you move the callback to a new line - don't forget to use round braces
```suggestion
  return ages.reduce((sum, age) => (
    sum + age
  )) / ages.length;
```